modules/ipi-install-bmc-addressing-for-dell.adoc

```diff
@@ -0,0 +1,136 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id='bmc-addressing-for-dell_{context}']
+
+= BMC addressing for Dell
+
+The `address` field for each `bmc` entry is a URL for connecting to the {product-title} cluster nodes, including the type of controller in the URL scheme and its location on the network.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: <host name>
+        role: <master | worker>
+        bmc:
+          address: <address>
+          username: <user>
+          password: <password>
+----
+
+For Dell hardware, Red Hat supports Redfish virtual media, Redfish network boot, and IPMI.
+
+.BMC address formats for Dell hardware
+[frame="topbot",options="header"]
+|====
+|Protocol|Address Format
+|Redfish virtual media| `idrac-virtualmedia://<out-of-band-ip>/redfish/v1/Systems/System.Embedded.1`
+|Redfish network boot|`redfish://<out-of-band-ip>/redfish/v1/Systems/System.Embedded.1`
+|IPMI|`ipmi://<out-of-band-ip>`
+|====
+
+[IMPORTANT]
+====
+Use `idrac-virtualmedia` as the protocol for Redfish virtual media. `redfish-virtualmedia` will NOT work on Dell hardware. Dell's `idrac-virtualmedia` uses the Redfish standard with Dell's OEM extensions.
+====
+
+See the following sections for additional details.
+
+.Redfish virtual media for Dell
+
+For Redfish virtual media on Dell servers, use `idrac-virtualmedia://` in the `address` setting. Using `redfish-virtualmedia://` will NOT work.
+
+ifeval::[{release} >= 4.6]
+ifeval::[{release} < 4.7]
+[NOTE]
+====
+Redfish virtual media on Dell servers has a known issue in {product-title} 4.6, which will be resolved in the 4.6.1 point release.
```

Comment:
I would just say "in a future release" instead of "the  4.6.1 point release" just in case of a release slip.